wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -193,6 +222,69 @@ var TranslationsPage = (function(window, $) {
       return container;
     };
 
+    this.filterLangs = function(list, filter, resource_filter, translationsData) {
+        var searchText = filter.value.toUpperCase();
+        var items = list.getElementsByTagName('li');
+        for (i = 0; i < items.length; i++) {
+            var item = items[i];
+            var langText = item.innerText.toUpperCase();
+            var displayItem = false;
+            // Check for name match
+            if (langText.indexOf(searchText) > -1) {
+                // Check for resource match
+                var resource_filter_value = resource_filter[resource_filter.selectedIndex].value.toLowerCase();
+                if (resource_filter_value === "all") {

```

Comment:
Inconsistent quotes with the rest of the file. Not a big deal in my opinion but just thought I would let you know in case you wanted to do something about that.

Reply:
Fixed.  Thanks.